test/services/bill-licences/fetch-bill-licence.service.test.js

```diff
@@ -0,0 +1,202 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const BillHelper = require('../../support/helpers/water/bill.helper.js')
+const BillModel = require('../../../app/models/water/bill.model.js')
+const BillRunHelper = require('../../support/helpers/water/bill-run.helper.js')
+const BillRunModel = require('../../../app/models/water/bill-run.model.js')
+const BillLicenceHelper = require('../../support/helpers/water/bill-licence.helper.js')
+const BillLicenceModel = require('../../../app/models/water/bill-licence.model.js')
```

Comment:
Think the bill-licence model and helper need to go above the bill-run model and helper

Reply:
They do!